### PR TITLE
Fixes custom config npm tut

### PR DESCRIPTION
**Pull request by mvanbrab**
What I needed to do to make this tutorial work to "expose my custom config as an npm package".

### CODE REVIEW

File: pages/docs/2_modify/1_getting_started/3_custom_init.md [hunk 1]
```diff
@@ -138,17 +135,21 @@ For this, add the following **scripts to our `package.json`** file:
     "prepublishOnly": "npm run build",
     "build:engine": "comunica-compile-config config/config-default.json > engine-default.js",
     "build:lib": "tsc",
-    "build": "npm run build:lib && npm run build:engine",
-    "postinstall": "npm run build"
```
Comment (rubensworks) [inline pages/docs/2_modify/1_getting_started/3_custom_init.md:142]: "postinstall" is wrong indeed, but we'll probably want to replace it by "prepare".

**Review by rubensworks — CHANGES_REQUESTED** [review]
Thanks @mvanbrab! Just one suggestion.

**Comment by mvanbrab** [comment]
If we add it as "prepare" script, we probably should remove the "prepublishOnly" script, also equal to "npm run build". See also <https://docs.npmjs.com/cli/v10/using-npm/scripts>

**Comment by rubensworks** [comment]
> If we add it as "prepare" script, we probably should remove the "prepublishOnly" script, also equal to "npm run build".

Yes, good idea!

**Comment by mvanbrab** [comment]
Added "prepare" script, removed "prepublishOnly".

**Comment by rubensworks** [comment]
Thanks @mvanbrab!

**Pull request merged**